documentation/modules/operators/proc-configuring-multiple-cluster-operators.adoc

```diff
@@ -0,0 +1,122 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-multiple-cluster-operators.adoc
+
+[id='assembly-configuring-proxy-config-cluster-operator-{context}']
+= Configuring Cluster Operator replicas
+
+[role="_abstract"]
+Increase the number of replicas and enable leader election to run additional Cluster Operator replicas in standby mode.
+To configure leader election, use leader election environment variables.
+
+To make the required changes, you can configure the following Cluster Operator installation files located in `install/cluster-operator/`: 
+
+* 060-Deployment-strimzi-cluster-operator.yaml
+* 022-ClusterRole-strimzi-cluster-operator-role.yaml
+* 022-RoleBinding-strimzi-cluster-operator.yaml
+
+Leader election has its own its own `ClusterRole` and `RoleBinding` RBAC resources that target the namespace where Cluster Operator is running, rather than the namespace it is watching. 
```

Comment:
```suggestion
Leader election has its own `ClusterRole` and `RoleBinding` RBAC resources that target the namespace where the Cluster Operator is running, rather than the namespace it is watching. 
```